sanitiser/_geo.js

```diff
@@ -46,26 +47,25 @@ function sanitize_bbox( clean, param ) {
     return;
   }
 
-  var bbox = [];
   var bboxArr = param.split( ',' );
 
   if( Array.isArray( bboxArr ) && bboxArr.length === 4 ) {
 
-    bbox = bboxArr.filter( function( latlon, index ) {
-      latlon = parseFloat( latlon, 10 );
-      return !(lat_lon_checks[(index % 2 === 0 ? 'lon' : 'lat')].is_invalid( latlon ));
+    var bbox = bboxArr.map( function ( latlon ){
+      return parseFloat( latlon, 10 );
+    });
+    bboxArr.forEach( function( latlon, index ) {
+      if( index % 2 === 1 && check_lat.is_invalid( latlon ) ){
+        throw new Error( 'Invalid lat: ' + latlon );
```

Comment:
[question] I see the unit tests pass, by what magic do we not also need `lng` validation in this block of code?